app/src/main/java/com/asfoundation/wallet/permissions/manage/view/PermissionsListFragment.kt

```diff
@@ -39,6 +38,15 @@ class PermissionsListFragment : BasePageViewFragment(), PermissionsListView {
   private lateinit var appInfoProvider: ApplicationInfoProvider
   private lateinit var permissionClick: BehaviorRelay<ApplicationPermissionViewData>
   private var toolbarManager: ToolbarManager? = null
+
+  private var _binding: FragmentPermissionsListLayoutBinding? = null
+  // This property is only valid between onCreateView and
+// onDestroyView.
+  private val binding get() = _binding!!
+
+  private val permissions_recycler_view get() = binding.permissionsRecyclerView
+  private val empty_state_view get() = binding.emptyStateView
```

Comment:
I think you don’t need to create values for that

Reply:
When initiating the binding by lazy we dont need a _binding and then a binding after.
Refer to OnboardingPaymentFragment for example to see its implementation